server/validators/goalTargetCompletionDateValidator.test.ts

```diff
@@ -11,52 +11,27 @@ describe('goalTargetDateValidator', () => {
     expect(errors).toStrictEqual(['Select a target completion date'])
   })
 
-  it('should validate given empty day', () => {
-    // Given
-    const day = ''
-    const month = '02'
-    const year = '2040'
-    // When
-    const errors = validateTargetDate('another-date', day, month, year)
-
-    // Then
-    expect(errors).toStrictEqual(['Enter a valid date'])
-  })
-
-  it('should validate given empty month', () => {
-    // Given
-    const day = '26'
-    const month = ''
-    const year = '2040'
-    // When
-    const errors = validateTargetDate('another-date', day, month, year)
-
-    // Then
-    expect(errors).toStrictEqual(['Enter a valid date'])
-  })
-
-  it('should validate given empty year', () => {
-    // Given
-    const day = '26'
-    const month = '02'
-    const year = ''
-    // When
-    const errors = validateTargetDate('another-date', day, month, year)
-
-    // Then
-    expect(errors).toStrictEqual(['Enter a valid date'])
-  })
-
-  it('should validate given an invalid date', () => {
-    // Given
-    const day = '26'
-    const month = '40'
-    const year = '2024'
-    // When
-    const errors = validateTargetDate('another-date', day, month, year)
-
-    // Then
-    expect(errors).toStrictEqual(['Enter a valid date'])
+  Array.of(
```

Comment:
I've removed a load of individual tests from this test class, and replaced them with an array
IE. they are all testing the same outcome; and are just a series of examples that should drive the same behaviour/outcome
All the tests I've removed are in this array, plus a few extras that I found based on the testing I did on the deployed app in order to reproduce the bug